drivers/tee/optee/call.c

```diff
@@ -144,10 +145,14 @@ u32 optee_do_call_with_arg(struct tee_context *ctx, phys_addr_t parg)
 	while (true) {
 		struct arm_smccc_res res;
 
+		optee_bm_timestamp();
```

Comment:
I think we shall in the future discuss whether to have this here by default or not. There are pros and cons to actually have the timestamp calls pre populated in the code. It's nice to have "some" default data when enabling the framework. But if I compare with ftrace for example, there is nothing printed/save just by enabling the feature. What will be measured depends on what the user want to see. But again, let's take that in future patches.